runtime/outer-PEC.js

```diff
@@ -101,11 +108,22 @@ class OuterPEC extends PEC {
           }
           handle.mapToStorage(fromHandle);
         }
+        let normalized = false;
         if (missingHandles.length > 0) {
-          error = `Recipe couldn't load due to missing handles [recipe=${recipe0}, missingHandles=${missingHandles.join('\n')}].`;
-        } else {
+          // Attempt to resolve missing handles with the strategizer.
+          const resolvedRecipes = await OuterPEC._applyResolveRecipe(this._arc, recipe0);
+          if (resolvedRecipes.length == 0) {
+            error = `Recipe couldn't load due to missing handles [recipe=${recipe0}, missingHandles=${missingHandles.join('\n')}].`;
+          } else {
+            assert(resolvedRecipes.length == 1, `Strategizer shouldn't produce more than one recipe with a single input recipe.`);
```

Comment:
i believe, it could produce more than 1 plan, if handles have tags, and not ids.

Reply:
If you're only using resolve-recipe, then definitionally only a single recipe should ever be generated.